### DISCUSSION TITLE
[Event]Happy 1024！

### DISCUSSION THREAD

**Discussion by ssusieee** [apache/doris#13586] [RFC]
Today is the 1024 Programmer's Day. On this special day, the community would like to thank all the programmers and the developers and contributors of the Apache Doris community. **Today, the community launched the theme event "This 1024, I want to talk to Apache Doris", inviting you to tell the story with Doris and your expectations for the future of Doris!**

Maybe you are just a novice who has just joined the community, full of curiosity and thirst for knowledge; maybe you are already a senior user of the community and have accompanied Doris every moment of growth; maybe you are the core backbone of the community, and there are traces of your dedication everywhere in Doris .....We welcome your comments here!

We will select 10 excellent expressors from the small partners who have successfully responded to the post to send our exquisite community gifts. In addition, we will select high-quality content creators for exclusive interviews!

### You can post the following types of replies here:
**1. My story with Apache Doris**

> The number of words is not less than 100 words, the expression is clear, smooth and fluent, and the content is deep
> Join time: x years, x months
> Reasons for joining: self-study, company use, research and selection, etc.
> User experience: The first time you enter the community and your current feelings about the community, which can be explained in terms of activity, support, etc.
> The biggest gain: can be explained in terms of ability, work, social interaction, etc.
> Community Message: Suggestions, Expectations, Future Development

**2. I hope Doris can develop like this in the future**

> There are no restrictions on the number of words and formats, and the expression is clear and well-founded. Please refer to:
> Doris future planning: add xx, improve xx..... to solve xxx
> Community building: increase xx input, carry out xx activities.....

**3. I share my experiences with members of the Doris community**
You have been in contact with Doris for a while. In the process of using it, what experience do you have to share with you? It can be learning diary, application practice, technical analysis.... Looking forward to your sharing and learning and progress with community students.
Due to the short duration of the event, please reply in the format below, and we will contact you to complete the content together.

> Topics:
> Outline: (No less than 3 points)
> E-mail Address:

This event ends at 23:59 on October 28th.

今天是1024 程序员节，在这个特别的日子里，社区要感谢所有的程序员，感谢 Apache Doris 社区的开发者及贡献者。**借此次 1024，社区发起「这个 1024，我想对 Apache Doris 说」主题活动，邀请你讲讲与 Doris 相知相识的故事，谈谈你对 Doris 未来的期望与畅想！**

也许你只是刚刚加入社区的小白，充满好奇与求知欲；也许你已是社区资深用户，陪伴了 Doris 的每一刻成长；也许你是社区的核心骨干，Doris 每个地方都有你倾注心血的痕迹.....我们欢迎你在此贴畅所欲言，一吐为快！

我们将从成功回帖的小伙伴中，**抽取 10 位优秀的“表达者”送出我们的精美社区盲盒**，另外，我们将选出优质内容创作者进行**独家专访**，助你成为社区中最亮的那颗星！

### 你可以在这里发表以下类型回帖：
**1. 我与 Apache Doris 相遇相知的故事**
（字数不少于100字，表述清晰，通顺流畅，阐述有逻辑，内容有深度是加分项哦！）
> 格式不做限制，建议但不限于以下内容：
> 加入多久：x年、x月
> 加入原因：自主学习、公司使用、调研选型等
> 印象及感受：初入社区及当前对社区的感受，可以从活跃度、支持度等方面阐述
> 最大的收获：可以从能力、工作、社交等方面阐述
> 社区寄语：建议、期待、未来发展

**2. 我希望 Doris 未来可以这样**

> 字数及格式不做限制，表达清晰，有理有据，可参考：
> Doris 规划：新增xx、提升xx .....，以解决xxx
> 社区建设：增加xx投入、开展xx活动.....

**3. 我来给 Doris 社区同学分享经验**
（想必你已经接触 Doris 有一段时间了，在使用的过程中，你一定有经验想和大家分享呢，可以是学习日记、可以是应用实践、也可以是技术解析....期待你的分享，与社区同学一起学习进步。）

> 因活动时间较短，请按照下方格式将想要分享的内容回复在此贴下，我们将与您主动联系，共同完成经验分享。
> 我想分享的内容是：
> 主题：
> 大纲：（不少于3点）
> 邮箱地址：

本次活动截止到 10月28日 23:59。

**Comment by dinggege1024** [comment]
加入多久：1个月(2022.08至今)
加入原因：公司使用、调研选型等
印象及感受：社区微信群很活跃，问题反馈处理比较快
最大的收获：开始尝试接触开源提交代码
社区寄语：跟社区和doris一起成长！

选型过程：
  vs TiDB：Doris在【动态分区管理】、【存储成本】、【OLAP（ROLLUP TiDB暂时不会支持）】上明显优于TiDB，目前已经规划从TiDB迁移全量数据的计划
TiDB VS Doris
* 动态分区&冷热存储：
    * Doris：完善易用（0额外开发成本）
    * TiDB：6.0支持冷热存储，但都需要额外开发维护
* OLAP
    * Doris：支持ROLLUP，BITMAP等易用操作
    * TiDB：TiFlash没有索引，查询都是全表查询到内存，不能单纯的列式存储，OLAP性价比低

总结：本次选型目的是降本增效，Doris可以为我们降低存储成本，同时也提升大量分析查询的效率

未来：目前正在铺开Doris的落地和使用，后面会总结更详细的内容

**Reply by ssusieee** [reply]
Great! Looking forward to seeing you everywhere in the community

**Reply by dinggege1024** [reply]
社区建设： 希望能在【广州】举行一次线下meetup

**Reply by ssusieee** [reply]
感谢你参加 1024 特别活动，恭喜你获得精美社区盲盒一份！
请你尽快在此处回复你的电子邮箱地址，我们将通过电子邮箱收集领奖信息！
为避免您个人信息泄露，请勿在此直接回复联系方式及地址！

**Reply by dinggege1024** [reply]
> 感谢你参加 1024 特别活动，恭喜你获得精美社区盲盒一份！ 请你尽快在此处回复你的电子邮箱地址，我们将通过电子邮箱收集领奖信息！ 为避免您个人信息泄露，请勿在此直接回复联系方式及地址！

dinggege1024@gmail.com

**Comment by LCehoennardo** [comment]
加入多久：2个月
加入原因：缘起于为公司调研，后面自己要学习因此接触越来越多
印象及感受：Doris很强大，社区也很活跃，大部分问题都能够在这里得到解答
最大的收获：学习到Doris，体会到开源社区的热情，感受到开源的魅力
社区寄语：一起成长，希望Doris越来越强大，社区越来越好！

**Reply by ssusieee** [reply]
Growing together！

**Reply by ssusieee** [reply]
感谢你参加 1024 特别活动，恭喜你获得精美社区盲盒一份！
请你尽快在此处回复你的电子邮箱地址，我们将通过电子邮箱收集领奖信息！
为避免您个人信息泄露，请勿在此直接回复联系方式及地址！

**Reply by LCehoennardo** [reply]
> 感谢你参加 1024 特别活动，恭喜你获得精美社区盲盒一份！ 请你尽快在此处回复你的电子邮箱地址，我们将通过电子邮箱收集领奖信息！ 为避免您个人信息泄露，请勿在此直接回复联系方式及地址！

nothingwrong73@outlook.com

**Comment by yz-jayhua** [comment]
**1. 我与 Apache Doris 相遇相知的故事**
**时间：** 2022年1月
**原因：** 公司使用&调研选型
**历程：**
一开始觉得apache doris就是个对标clickhouse的普通mpp数据库；
而后，公司中高层决定选用apache doris作为这边中台底座db，于是开始规划功能场景预研、对比测验、压测调优和上线各阶段的方案和WBS；
其中压测调优阶段前期让我有些质疑这玩意儿是怎么火起来的？？？

> 当时用的0.15版本，10个并发1T的TPCDS场景一压be就oom crash

抱怨是没有意义的，于是开始想办法与社区搭上线；
缘分让我们相遇，非常感谢社区群让我结识到锋哥（张家锋）、率哥（曲率）和强哥（杨勇强）等社区&selectDB的前辈们；
于是开始向他们请教各种功能和性能的问题，锋哥也非常大无畏地单独拉了个技术支持群，期间也不记不清拉了多少次线上会议、battle了多少回bug事项以及问了多少次有些白痴的问题（官文一搜就能查到）；
好在，锋哥他们都比较耐心和负责，基本上大部分问题都非常给力地解答和解决了。

> 前面说的压测调优阶段前期让我质疑的事儿，后续版本apache doris的健壮性越来越好，be oom crash的情况得到了非常大的改善

因为大半年都在不断地测验apache doris和建设数据中台底座，也没太多时间去读它的源码和了解其前世今生，现在这边也基本上线发布到生产运行了，虽然还存在些大大小小的问题，但也算是有些喘息的机会去多看看apache doris。
当我开始对比apache doris与其它同源db、了解它的过去时，我发现：嗯，它的背景很有主角范儿；

> 开源无私/低调务实/潜力实力并存

当我开始积极地参与apache doris线上的一些分享、了解到它的未来规划时，突然觉着它有点巨人的感觉、生态规划的很好，无论对于数据湖的集成、各种load场景还是etl场景的蓝图都有，奈何我文化水平不高、只能吆喝一声：真TMNB！！！
后续也希望自己能够参与到apache doris的建设中，贡献绵薄之力。


**2. 我希望 Doris 未来可以这样**
**Doris 规划：**
1. bucket动态化（大部分初入apache doris的小伙伴应该都踩过tablet设置不合理的坑吧^^）
2. 增加队列机制&etl场景

**社区建设：** **希望能在深圳举行一次线下meetup**


**3. 我来给 Doris 社区同学分享经验**
我想分享的内容是：
主题：apache doris的应用场景
大纲：
1. 理想化与现实中的流批一体on apache doris
2. apache doris在跨境物流场景中的优势与不足
4. 未来展望与规划
邮箱地址：zhbinhua@vip.qq.com

**Reply by ssusieee** [reply]
Nice！I like your comment and look forward to your sharing！

**Reply by yz-jayhua** [reply]
Think! I'm looking forward apache doris grow  more and more powerful!

**Reply by ssusieee** [reply]
> Think! I'm looking forward apache doris grow more and more powerful!
hi，我们想邀约您进行个人专访及用户案例共创，相关信息已于昨日发送到您的邮箱，辛苦查收回复~

**Comment by Yukang-Lian** [comment]
**1. 我与 Apache Doris 相遇相知的故事**

加入多久：3个月
加入原因：通过参加开源之夏活动与Doris相识
印象及感受：Doris社区非常活跃，对新人很友好，大佬都会耐心的回答问题
最大的收获：了解了团队协作的流程，提高了编程技术，结识了志同道合的小伙伴们
社区寄语：期望Doris能够吸纳越来越多的小伙伴，与Doris共同成长

**2. 我希望 Doris 未来可以这样**

Doris 规划：多增加good first issue，让新人也能快速融入社区
社区建设：增加对于学生团体的宣传力度，让更多的学生了解Doris，使用Doris，为Doris做贡献

**Reply by ssusieee** [reply]
感谢你参加 1024 特别活动，恭喜你获得精美社区盲盒一份！
请你尽快在此处回复你的电子邮箱地址，我们将通过电子邮箱收集领奖信息！
为避免您个人信息泄露，请勿在此直接回复联系方式及地址！

**Comment by yqzdy1207** [comment]
1. 我与 Apache Doris 相遇相知的故事
时间： 2018年（记不太清了）
原因： 了解
历程：
a.从0.13发版认识、1.0正式使用,公司内版本由我个人来部署和升级，目前线上随发版而升级，已经1.1.3了，非常期待1.2的功能
b.期待商业化selectDB在移动云上合作，到时候就可以更加深度的使用、学习和合作了
c.社区群内非常的活跃，免费&专业&及时的答疑
d.看到非常多的公司因为Doris的出现简化了架构、涵盖了离线/实时数仓,在多业务领域都在有最佳实践

2. 我希望 Doris 未来可以这样
提升稳定性、考虑周边生态组件，将数据治理和数据质量的功能吸纳

**Reply by ssusieee** [reply]
感谢你参加 1024 特别活动，恭喜你获得精美社区盲盒一份！
请你尽快在此处回复你的电子邮箱地址，我们将通过电子邮箱收集领奖信息！
为避免您个人信息泄露，请勿在此直接回复联系方式及地址！

**Comment by zoujianchao** [comment]
**1. 我与 Apache Doris 相遇相知的故事**

**加入多久：** 2022年10月
**加入原因：** 调研选型&公司使用
**印象及感受：** 社区很活跃,官方文档写的很好,活动直播内容很不错
**最大的收获：** 社区群里每天都可以看到不同的问题被解答,学到很多; 技术有成长
**社区寄语：** 感觉现在的版本已经满足, 期待1.1.3版本之后的Apache Doris能更优秀

**2. 我希望 Doris 未来可以这样**

**Doris 规划：** bucket动态化; 提升稳定性; (希望做大做强:融入数仓,数分相关组件,有自己的报表平台,调度平台等)

**Reply by ssusieee** [reply]
感谢你参加 1024 特别活动，恭喜你获得精美社区盲盒一份！
请你尽快在此处回复你的电子邮箱地址，我们将通过电子邮箱收集领奖信息！
为避免您个人信息泄露，请勿在此直接回复联系方式及地址！
